src/style.css

```
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  display: inline-flex;
+  flex-flow: column;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--5 .stars__star,
```

do it same way as all other
```suggestion
.stars--5 .stars__star:nth-child(-n + 5),
```